spec/services/candidate_interface/section_policy_spec.rb

```diff
@@ -51,6 +51,30 @@
       end
     end
 
+    context 'when candidate submitted in the 5 working days window' do
+      let(:current_application) { create(:application_form, submitted_at: 1.day.ago) }
+
+      before do
+        create(:application_choice, :awaiting_provider_decision, application_form: current_application)
+      end
+
+      context 'when accessing an editable section' do
+        let(:controller_path) { 'candidate_interface/personal_details/review' }
+
+        it 'returns true' do
+          expect(section_policy.can_edit?).to be true
+        end
+      end
+
+      context 'when accessing an non editable section' do
```

Comment:
`a non editable` small typo. And this spec reads a bit weird. A non editable section is editable. Meaning it's not really non editable.

Reply:
Yes that was difficult to explain. Will be a future non editable after the 5 day window. 🤔 "when accessing a future non editable section" is better?

Reply:
I think it is, yeah.

Reply:
Done